### DISCUSSION TITLE
Is the top-level "id" property the GERS ID for a place?

### DISCUSSION THREAD

**Discussion by thisisaaronland** [OvertureMaps/data#18] [Discussion]
For example when I do something like:

```
COPY (SELECT type, id, confidence ...) 
```

From the `places` data files I see that the output contains an `id` property like this:

```
{ "type": "Feature", "properties": { "type": "place", "id": "tmp_BD353FB8A24BC4F580CCC77309DBBDC3",
```

But it is unclear if this is the GERS or why the identifier is prefixed with "tmp_"

**Comment by varapmsft** [comment]
As mentioned in the [release notes](https://overturemaps.org/download/overture-july-alpha-release-notes/), these are placeholder GERS Ids for now, hence they are prefixed with "tmp_". In future releases, the official GERS IDs will be introduced gradually.

_NOTE: Placeholder GERS IDs are populated in the data. These are needed to populate the Context property which points to another admin entity. For example, the Context property of Washington state feature has the GERS ID of the United States entity as value. This may not be representative of the final GERS format and should not be the basis for long-term development work._

**Comment by thisisaaronland** [comment]
Thanks. Does anyone know whether a key / crosswalk / lookup table will be provided for mapping temporary IDs to their final GERS ID value?
